var/spack/repos/builtin/packages/procps/package.py

```diff
@@ -17,14 +18,16 @@ class Procps(AutotoolsPackage):
     version("master", branch="master")
     version("3.3.15", tag="v3.3.15")
 
+    variant("nls", default=True, description="Enable Native Language Support.")
+
     depends_on("autoconf", type="build")
     depends_on("automake", type="build")
     depends_on("libtool", type="build")
     depends_on("m4", type="build")
     depends_on("pkgconfig@0.9.0:", type="build")
     depends_on("dejagnu", type="test")
     depends_on("iconv")
-    depends_on("gettext")
+    depends_on("gettext", when="+nls")
```

Comment:
(question) Was this made optional on account of some additional issues? I don't immediately see how introducing the variant is part of addressing the problems this PR intends to solve.

Reply:
The rest of the improvements to this recipe provide the consistency: this just removes an unnecessary dependency on `gettext` when native language support is not required.